cprover_bindings/src/goto_program/expr.rs

```diff
@@ -41,6 +50,7 @@ pub struct Expr {
     value: Box<ExprValue>,
     typ: Type,
     location: Location,
+    size_of_annotation: Option<Type>,
```

Comment:
Why do we need an additional `Option<Type>` here? We already have the type in `typ`, and if I'm not mistaken this hint is something you always want to have for aggregate types.

Reply:
No, the annotation really says (and I have now expanded the comment to say this): "... the expression is the result of computing `size_of(type)`."